### DISCUSSION TITLE
Provisioned token lacks permissions for newly created repositories

### DISCUSSION THREAD

**Discussion by anoblet** [community/community#30444] [Codespaces]
When using `$GITHUB_TOKEN` to install a  `composer` package from a private organizational repository, the token lacks permissions for repositories that were created recently.

We have multiple private packages hosted inside of a Github organization, though it seems only the newest one throws 403. Replacing the provisioned `$GITHUB_TOKEN` with a personal access token with `repo` privileges allows the repository to be cloned. This breaks our automation for setting up workspaces for new team members.

**Comment by bdmac** [comment]
Depending on how your devcontainer is set up you may be able to add something like this:
```
  "customizations": {
    "codespaces": {
      "repositories": {
        "your-org/*": {
          "permissions": {
            "contents": "read",
            "packages": "read"
          }
        }
```

**Reply by anoblet** [reply]
I just tried this. No luck :/
